app/controllers/application_controller/ci_processing.rb

```diff
@@ -1619,6 +1619,42 @@ def scanhosts
     host_button_operation('scan', 'Analysis')
   end
 
+  # Scan all selected or single displayed image(s)
+  def scanimages
```

Comment:
Well, I didn't saw vm_button_opertaions when I did this. I've looked at the host model.
It seems that there are multiple places in which a similar logic is reached, but each place have some difference which make it a bit harder to unite, and I wouldn't like to make this PR a complete refactor of vm_button/host button/etc.